host-interaction/driver/disable-driver-code-integrity.yml

```diff
@@ -0,0 +1,22 @@
+rule:
+  meta:
+    name: disable driver code integrity
+    namespace: host-interaction/driver
+    author: william.ballenthin@fireeye.com
+    scope: function
+    references:
+      - https://www.fuzzysecurity.com/tutorials/28.html
+      - https://j00ru.vexillium.org/2010/06/insight-into-the-driver-signature-enforcement/
+    examples:
+      - 31CEE4F66CF3B537E3D2D37A71F339F4:0x140004070
+  features:
+    - or:
+      - string: "CiInitialize"
+        description: exported symbol name used to resolve code integrity configuration
```

Comment:
public techniques use memory scanning near this symbol to find the kernel address of the flag and flip it. i'd hoped to use some `offset` features but there's not too much to work with.

most public techniques abuse vulnerable drivers to r/w kernel memory from user code.